ios/FluentUI/DrawerVnext/Drawer+Modifiers.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+public extension Drawer {
```

Comment:
Same thing here regarding the need for a separate file and an extension: I think this should go to the Drawer file as a func and the extension should be deleted unless we have more modifiers to justify it.

Reply:
I have used extension when there is a logical grouping of functionalities, modifiers are going to be extended as we add vertical drawer. The extensions in Drawer are split based on UIKit/SwiftUI native interfaces. As the drawer will grow there would be necessity to clarify the distinction. A client might end up using the other interface if there is not sufficient clear distinction between them. we can discuss this more offline if required

Reply:
I can see the value of creating separate files for modifiers related to a certain functionality.
My understanding of your idea in this case would be something like Drawer+Delegate.swift or Drawer+State.swift.
Drawer+Modifiers.swift seems like a file that will contain all modifiers for it irrespective of their functionality group.
We can indeed discuss offline.

Reply:
I will remove the drawer modifier for now as it contains only single API. We can discuss this later as we extend the functionality and more API's in drawer